nupic/data/fieldmeta.py

```diff
@@ -57,6 +59,27 @@ class FieldMetaInfo(FieldMetaInfoBase):
   3.
   """
 
+
+  def __init__(self,
+               name,
+               type,  # pylint: disable=W0622
```

Comment:
@oxtopus: unfortunately, the `type` arg was used in the legacy implementation, and changing it would be out of scope for this task. Take a look at the base class definition above it `FieldMetaInfoBase = namedtuple('FieldMetaInfoBase', ['name', 'type', 'special'])`. This may have been done by me (?) as I was learning the pythonic ways and before I knew better.